app/komponenter/språkvelger/språkvelger.module.css

```diff
@@ -1,8 +1,8 @@
-.språkvelger {
+.selectKonteiner {
   width: 10.75rem;
 }
 
-.label {
+.labelKonteiner {
```

Comment:
Denne kunne også vært bare `label` fordi den brukes for å lage labelen og det er ingen andre labels i språkvelgeren, men den er til spanen rundt så den kan også ses på som en kontainer. 

Her kan dere velge selv

Reply:
Endret det. 

Se commit : 8291bbc